libraries/AC_AttitudeControl/AC_AttitudeControl_Heli.h

```diff
@@ -127,7 +136,7 @@ class AC_AttitudeControl_Heli : public AC_AttitudeControl {
     float _passthrough_yaw;
 
     // get_roll_trim - angle in centi-degrees to be added to roll angle. Used by helicopter to counter tail rotor thrust in hover
-    float get_roll_trim_rad() override { return constrain_float(radians(_hover_roll_trim_scalar * _hover_roll_trim * 0.01f), -radians(10.0f),radians(10.0f));}
+    float get_roll_trim_rad() override { return constrain_float(radians(get_roll_trim_cd() * 0.01f), -radians(10.0f),radians(10.0f));}
```

Comment:
```suggestion
    float get_roll_trim_rad() override { return radians(get_roll_trim_cd() * 0.01); }
```
`get_roll_trim_cd()` applies the constrain already.